packages/device_info_plus/device_info_plus/pubspec.yaml

```
@@ -25,7 +25,7 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  device_info_plus_platform_interface: ^2.3.0
+  device_info_plus_platform_interface: 2.3.1
```

```suggestion
  device_info_plus_platform_interface: ^2.3.1
```
melos CI check is failing because it expects to use caret syntax